src/VisualStudio/DevKit/Impl/Microsoft.VisualStudio.LanguageServices.DevKit.csproj

```diff
@@ -45,18 +44,20 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.DevKit.UnitTests" />
   </ItemGroup>
-  
-  <!--
-  Add these additional runtime dependencies to our NuGet package.
-  -->
-  <Target Name="CollectPackInputs" DependsOnTargets="Build" BeforeTargets="Pack" Returns="@(Content)">
+
+  <ItemGroup>
+    <Content Include="$(PkgMicrosoft_VisualStudio_Telemetry)\lib\netstandard2.0\Microsoft.VisualStudio.Telemetry.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="$(PkgMicrosoft_VisualStudio_RemoteControl)\lib\netstandard2.0\Microsoft.VisualStudio.RemoteControl.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="$(PkgMicrosoft_VisualStudio_Utilities_Internal)\lib\netstandard2.0\Microsoft.VisualStudio.Utilities.Internal.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="$(PkgSystem_Configuration_ConfigurationManager)\lib\netstandard2.0\System.Configuration.ConfigurationManager.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="$(PkgMicrosoft_VisualStudio_Debugger_Contracts)\lib\netstandard2.0\Microsoft.VisualStudio.Debugger.Contracts.dll" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="$(TargetPath)" Pack="true" PackagePath="content" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
+
+  <!-- Used by LanguageServer.UnitTests project to be able to copy all the content to its own directory -->
+  <Target Name="GetPackInputs" DependsOnTargets="Build" Returns="@(_Content)">
     <ItemGroup>
-      <Content Include="$(PkgMicrosoft_VisualStudio_Telemetry)\lib\netstandard2.0\Microsoft.VisualStudio.Telemetry.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(PkgMicrosoft_VisualStudio_RemoteControl)\lib\netstandard2.0\Microsoft.VisualStudio.RemoteControl.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(PkgMicrosoft_VisualStudio_Utilities_Internal)\lib\netstandard2.0\Microsoft.VisualStudio.Utilities.Internal.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(PkgSystem_Configuration_ConfigurationManager)\lib\netstandard2.0\System.Configuration.ConfigurationManager.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(PkgMicrosoft_VisualStudio_Debugger_Contracts)\lib\netstandard2.0\Microsoft.VisualStudio.Debugger.Contracts.dll" Pack="true" PackagePath="content" />
-      <Content Include="$(TargetPath)" Pack="true" PackagePath="content" />
+      <_Content Include="@(Content)" />
```

Comment:
Should we only include items such that `%(Content.Pack) == "true"`?

Reply:
done!